Update URL for static deploy of Wordpress site (#1999)
cloudflare/cloudflare-docs: products/workers/src/content/platform/sites/start-from-existing.md

- [Gatsby](https://www.gatsbyjs.org/docs/quick-start/), requires Node
- [Jekyll](https://jekyllrb.com/docs/), requires Ruby
- [Eleventy](https://www.11ty.io/#quick-start), requires Node

> - [WordPress](https://wordpress.org) (see our tutorial on [deploying static WordPress sites with Workers](/tutorials/deploy-a-static-wordpress-site))

Reviewer: ```suggestion
- [WordPress](https://wordpress.org) (refer to the tutorial on [deploying static WordPress sites with Workers](/tutorials/deploy-a-static-wordpress-site))
```
